### PR TITLE
feat: add across v2

### PR DESCRIPTION
**Pull request by liuyepiaoxiang**
## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [x] (optional) As a contributor, my Ethereum address/ENS is:lious.eth
- [x] (optional) As a contributor, my Twitter handle is:@jason_lious

### CODE REVIEW

**Review by immasandwich — COMMENTED** [review]
We don't support Boba yet, so unfortunately, this can't be added. Let's remove those files before being able to merge this.

File: src/apps/across-v2/across.pool.definitions.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    chainId: 42161,
+    isSpokePool: true,
+  },
+];
+
+export const POOL_TOKENS:AcrosstTokenDefinition[] = [
+  {
+    queryKey: 'badger',
+    symbol: 'BADGER',
+    tokenAddress: '0x3472a5a71965499acd81997a54bba8d852c6e53d',
```
Comment (immasandwich) [inline src/apps/across-v2/across.pool.definitions.ts:52]: This won't work... the hub and spoke pools are on different networks, so the addresses of the tokens that can be deposited are different on each network.

You'll need to create a configuration of tokens _per network_.

**Comment by immasandwich** [comment]
Hey @liuyepiaoxiang, sorry for the delay here. I'll review this today or tomorrow and we'll try to push it along faster!

**Comment by SamIam-0x** [comment]
(Will let @immasandwich handle the PR)

Hello @liuyepiaoxiang !

Looks like you did not provide an address when submitting your PR. I recognize it may have seemed kind of random that we were asking for a wallet address with a PR submission, but we are actually looking for ways to reward Studio contributors in the future. 

Is there an address you’d want to provide? Could even be an empty address, if you do not have an doxxed address

Let me know! Also, if you’d like to privately provide it, feel free to shoot me a TG https://t.me/SamIAm_0x or DM on the Zapper Discord.

Thanks!
Sam

**Comment by immasandwich** [comment]
Yep we can merge this and I can follow-up with some updates in the code (our patterns have changed a lot since this PR was opened)

**Comment by hlambur** [comment]
@immasandwich it looks like your merge failed. Anything I can do (or that I can ask @liuyepiaoxiang to help with) to get this done. I'm an impatient Zapper user :)

**Comment by SamIam-0x** [comment]
Unrelated to the branch stuff, please do let me know if you want to provide an address @liuyepiaoxiang ! Thanks!

**Comment by liuyepiaoxiang** [comment]
OK,I will follow your patterns to change it

**Comment by hlambur** [comment]
Any update on this @SamIam-0x @liuyepiaoxiang ?

I'd really love to use this!

**Comment by immasandwich** [comment]
Coming in https://github.com/Zapper-fi/studio/pull/1612

**Comment by immasandwich** [comment]
I mean https://github.com/Zapper-fi/studio/pull/1613

**Pull request closed without merging**